addon-test-support/audit.js

```diff
@@ -44,6 +44,18 @@ function isPlainObj(obj) {
   return typeof obj == 'object' && obj.constructor == Object;
 }
 
+/**
+ * Determines whether supplied object contains `include` and `exclude` axe
+ * context selector properties. This is necessary to distinguish an axe
+ * config object from a context selector object, after a single argument
+ * is supplied to `runA11yAudit`.
+ * @param {Object} obj
+ * @return {Boolean}
+ */
+function isNotSelectorObj(obj) {
+  return !obj.hasOwnProperty('include') && !obj.hasOwnProperty('exclude');
```

Comment:
Just want to double check that both `include` and `exclude` will always be present, right?

Reply:
We can use either/or. The possible combinations can be just `include`, just `exclude`, both `include` and `exclude`, or neither one. So this particular conditional will return true if neither is present, but false if either one or both are detected. I have tested all the permutations to make sure this behaves as expected. Let me know if you have additional questions. Thanks!

Reply:
That works for me! Great stuff 😃